internal/rbd/rbd_util.go

```diff
@@ -96,6 +96,11 @@ type rbdImage struct {
 	// VolSize is the size of the RBD image backing this rbdImage.
 	VolSize int64
 
+	// image striping configurations.
+	StripCount uint64
+	StripUnit  uint64
+	ObjectSize int64
```

Comment:
Why is `ObjectSize` signed, while `StripeUnit` and `StripeCount` are unsigned?

Reply:
I was thinking order and objectSize both are the same took it from here https://github.com/ceph/go-ceph/blob/master/rbd/rbd.go#L153-L155. after looking at https://github.com/ceph/ceph/blob/master/src/librbd/api/Image.cc#L577-L587 keeping it as unit64